test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -695,26 +701,35 @@ func startRealMasterOrDie(t *testing.T, certDir string) (*allClient, clientv3.KV
 				t.Log(err)
 			}
 
-			time.Sleep(100 * time.Millisecond)
+			time.Sleep(time.Second)
 		}
 	}()
 
-	if err := wait.PollImmediate(100*time.Millisecond, 10*time.Second, func() (done bool, err error) {
```

Comment:
This both increases the total time to wait and decreases the rate of polling.

Why decrease the rate of polling? Won't this just mean there is more potential for this to take longer to run than it needs to?

Reply:
> This both increases the total time to wait and decreases the rate of polling.

I attempt to start a very real-ish master, which can take longer to report healthy.  I felt like it was counterproductive to keep asking the master if it was healthy.  The total time only matters in cases where the master fails to start (meaning something else is broken).

> Why decrease the rate of polling? Won't this just mean there is more potential for this to take longer to run than it needs to?

Taking ~1 second longer isn't really an issue IMO (especially with parallel integration tests).